.github/workflows/node-pr.yml

```diff
@@ -39,7 +39,9 @@ jobs:
           shopt -s globstar # enable globstar option to use ** for recursive matching
           for lockfile in samples/**/yarn.lock; do # loop over all yarn.lock files
             dir=$(dirname "$lockfile") # get the directory of the lock file
-            echo "Running yarn install and yarn build for $dir"
-            yarn --cwd "$dir" install --frozen-lockfile # install dependencies
-            yarn --cwd "$dir" build # run build script
+            if [[ "$dir" != "samples/apps" ]]; then # exclude samples/apps directory
```

Comment:
We don't have a samples/apps/yarn.lock. Will $dir = "samples/apps"?

Reply:
Does running `yarn install` in the current working directory also install the dependencies defined in parent directories? Or would we have to run `yarn install` in both directories to install all the dependencies?

Reply:
it's currently the latter but it would be nice to have the former. I'll look into adding the workspace

Reply:
I looked into Yarn workspaces but it doesn't seem that adding one would enable the behavior you mentioned, and rather would allow us to have dependencies that need to exist in each app, be resolved from the root directory.
for us this isn't necessary, as the dependencies only need to exist in the root directory, since ESLint is smart enough on its own to resolve its dependencies from the directory that the `.eslintrc.js` lives in.

but regardless, at least from what I gathered in my limited research, `yarn install` would still need to be ran in both the root and subfolders.

Reply:
Should we update the ReadMes to reflect this change in the setup step?

Reply:
definitely, but not right now I don't think. I'm still making sure we'll have the correct setup so that we won't conflict with the default `react-scripts` ESLint config. right now, no changes in the setup are necessary since ESLint is disabled in the build.